post_installation_configuration/aws-compute-edge-tasks-zones.adoc

```diff
@@ -1,50 +1,51 @@
 :_mod-docs-content-type: ASSEMBLY
-[id="aws-compute-edge-tasks"]
-= AWS Local Zone tasks
+[id="aws-compute-edge-tasks-zones"]
+= AWS zone tasks
 include::_attributes/common-attributes.adoc[]
-:context: aws-compute-edge-tasks
+:context: aws-compute-edge-tasks-zones
+:zone-type: Local Zones or Wavelength Zones
 
 toc::[]
 
-After installing {product-title} on Amazon Web Services (AWS), you can further configure AWS Local Zones and an edge compute pool, so that you can expand and customize your cluster to meet your needs.
+After installing {product-title} on Amazon Web Services (AWS), you can further configure AWS {zone-type} and an edge compute pool, so that you can expand and customize your cluster to meet your needs.
 
-// Extend existing clusters to use AWS Local Zones
+// Extend existing clusters to use AWS zones
 include::modules/post-install-edge-aws-extend-cluster.adoc[leveloffset=+1]
 
 [role="_additional-resources"]
 .Additional resources
 
 * For more information about AWS Local Zones, the supported instances types, and services, see link:https://aws.amazon.com/about-aws/global-infrastructure/localzones/features/[AWS Local Zones features] in the AWS documentation.
+* For more information about AWS Local Zones, the supported instances types, and services, see link:https://aws.amazon.com/wavelength/features/[AWS Wavelength features] in the AWS documentation.
 
-
-// About the Edge Compute Pool
+// Edge compute pools and AWS zones
 include::modules/edge-machine-pools-aws-local-zones.adoc[leveloffset=+2]
 
 [id="post-install-extend-existing-to-local-zones-mtu"]
-=== Changing the Cluster Network MTU to support AWS Local Zones subnets
+=== Changing the Cluster Network MTU to support AWS zone subnets
 
 You might need to change the maximum transmission unit (MTU) value for the cluster
-network so that your cluster infrastructure can support Local Zone subnets.
+network so that your cluster infrastructure can support {zone-type} subnets.
 
 // About the cluster MTU
 include::modules/nw-cluster-mtu-change-about.adoc[leveloffset=+3]
 
-// Changing the cluster MTU
+// Changing the cluster MTU -- CONFIRM WITH MARCO as steps differ
```

Comment:
It's not completely correct, we don't need to tech users to change the hardware MTU, only cluster-network.

> Prepare your configuration for the hardware MTU:

So this rendered section have more items than needed: 
https://69537--ocpdocs-pr.netlify.app/openshift-enterprise/latest/post_installation_configuration/aws-compute-edge-tasks-zones#nw-cluster-mtu-change_aws-compute-edge-tasks-zones

Please review the original approach/page/task book:
https://docs.openshift.com/container-platform/4.14/post_installation_configuration/aws-compute-edge-tasks.html#post-install-extend-existing-to-local-zones-mtu

Reply:
Thanks, @mtulio . Should I filter out the [hardware MTU](https://docs.openshift.com/container-platform/4.14/post_installation_configuration/aws-compute-edge-tasks.html#post-install-extend-existing-to-local-zones-mtu:~:text=the%20MTU%20change.-,MTU%20value%20selection,-When%20planning%20your) info from the AWS edge zone tasks doc?

![Screenshot from 2024-01-23 11-28-23](https://github.com/openshift/openshift-docs/assets/57954076/d5b83f2d-3219-4c35-8304-f935544770be)